tasks/pipeline.py

```diff
@@ -6,6 +6,8 @@
 from collections import defaultdict
 from datetime import datetime
 
+import subprocess as sp
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏I'm usually trying to avoid using `subprocess` while writing invoke tasks because it's already supported by invoke and much easier to read / maintain. Instead of this you can use `ctx.run` which is done in the other tasks !

Reply:
yes will change this